### PR TITLE
chore: Use large GH hosted runner for stress test

### PR DESCRIPTION
**Pull request by bergundy**

### CODE REVIEW

File: .github/workflows/stress.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ env:
 
 jobs:
   stress-test:
-    runs-on: [self-hosted, Linux, X64]
+    runs-on: [ubuntu-latest-4-cores, linux]
```
Comment (lorensr) [inline .github/workflows/stress.yml:20]: so we're going to have 2 org runner groups with these names that this repo is given access to? why the second?

https://docs.github.com/en/actions/using-github-hosted-runners/using-larger-runners#overview-of-larger-runners

**Review by lorensr — APPROVED** [review]

File: .github/workflows/stress.yml (hunk 1 above)
Comment (bergundy) [reply .github/workflows/stress.yml:20]: Not sure I understand the question.
This is just requiring 2 labels, the `linux` label might be redundant.

**Review by Sushisource — APPROVED** [review]
9 min is not bad

**Comment by bergundy** [comment]
@Sushisource it's not 9 minutes, the runner isn't picking up the job for some reason.

**Comment by lorensr** [comment]
Is there a runner in the `temporalio` org with those 2 tags that this repo has access to?

**Comment by bergundy** [comment]
There's supposed to be one, I'm trying to understand why it's not starting.

**Comment by Sushisource** [comment]
15 minutes, less great, lol.  Still fine considering our check-in rate though I think.

**Pull request merged**